### DISCUSSION TITLE
Opiniones de practica Abogabot

### DISCUSSION THREAD

**Discussion by JoseEduardoVelazquezCN** [Launch-X-Latam/MisionFrontEnd#190] [General]
Hola espero sus opiniones, y críticas constructivas ✋ 
Link a mi repositorio: https://github.com/JoseEduardoVelazquezCN/Paracticas-Laucnch-X-Mision-FrontEnd

**Comment by maxkario** [comment]
Me gusta. Sobre todo tu UI para ambos casos. Me dio mas claridad de como llevar el diseño UI, ya que estaba en blanco respecto a eso.
Pienso que deberias crear un diseño UI tambien para la version web, piensa en el usuario que utiliza mas la web que su tlf.

**Reply by JoseEduardoVelazquezCN** [reply]
Gracias por tu observación✔️, me enfoque un poco más en el diseño para móvil  por qué siento que es más fácil pasar del tamaño móvil a web 
Y como era una especificación de las características de la práctica me enfoque más en eso 
Y analizando todo creo que hoy en día cualquiera tiene más a la mano un celular que una computadora

**Comment by EduGarcia502** [comment]
muy buen trabajo camarada lo felicito y gracias por compartir con nosotros tus pensamientos he ideas, saludos y muy bue trabajo

**Comment by AlbertoSF99** [comment]
Tu UI está muy bien llevado, me gustó mucho el diseño que le diste. Me surgió una duda, ¿la toma de datos puede elaborarse desde la misma documentación en github o debe subirse en un documento (Word) como el template que viene en la práctica?

**Reply by JoseEduardoVelazquezCN** [reply]
Gracias, sobre los requerimientos me parece que es como te agrade más, por ahí ví un debate que decía que podía ser como quisieras, no subí el documento por qué había muchos puntos que deje vacíos jajaja

**Comment by BrayanGil090720** [comment]
Hey bro excelente practica, como recomendación en la parte de inicio de sesión, puedes poner hay mismo el registrate ahora, y olvide la contraseña, la parte de contacto normalmente va en la sección del footer, de hay en mas te quedo excelente hermano
